tools/asset-sync/assets.ps1

```diff
@@ -0,0 +1,552 @@
+Set-StrictMode -Version 3
+
+$REPO_ROOT = Resolve-Path (Join-Path $PSScriptRoot ".." "..")
+$ASSETS_STORE = (Join-Path $REPO_ROOT ".assets")
```

Comment:
I assume once we start consuming this script these will become parameters or set in some way outside of this script.

Reply:
Well, this thing will be shipped as part of `eng/common` if we stick with pshell. With that being the case, no, those will actually stay. (if only as defaults for a param on the script 👍 )